GVFS/GVFS.Common/NamedPipes/NamedPipeMessages.cs

```diff
@@ -235,7 +235,8 @@ public enum Identifier
                 {
                     AutomountStart,
                     MountSuccess,
-                    MountFailure
+                    MountFailure,
+                    UpgradeAvailable

```

Comment:
Is anything sending the `UpgradeAvailable` notifications right now?  I could only find where `MountFailure` is being sent:

https://github.com/microsoft/VFSForGit/blob/27758788bc228371fffd0a6ba968e9ee17f61177/GVFS/GVFS.Service/RepoRegistry.cs#L186

How were you able to test this locally?

Reply:
`UpgradeAvailable` is not hooked up yet. For testing I use [this branch](https://github.com/alameenshah/VFSForGit/tree/notification_user_actions_notify_verb) in my fork - It has a `notify` verb that can send all supported notifications to the UI.